os/net/netdev/netdev_ioctl.c

```diff
@@ -1003,6 +1006,69 @@ static int netdev_rtioctl(FAR struct socket *sock, int cmd, FAR struct rtentry *
 }
 #endif							/* CONFIG_NET_ROUTE */
 
+#if CONFIG_NET_LWIP
+/****************************************************************************
+ * Function: lwip_func_ioctl
+ *
+ * Description:
+ *   Call lwip API
+ *
+ * Parameters:
+ *   sock     Socket structure
+ *   cmd      The ioctl command
+ *   arg      Type of the information to get
+ *
+ * Returned Value:
+ *   >=0 on success (positive non-zero values are cmd-specific)
+ *   Negated errno returned on failure.
+ *
+ ****************************************************************************/
+
+int lwip_func_ioctl(FAR struct socket *sock, int cmd, void  *arg)
+{
+	int ret = -EINVAL;
+	struct req_lwip_data *in_arg = (struct req_lwip_data *)arg;
+	if (!in_arg) {
+		return ret;
+	}
+	if (cmd != SIOCLWIP) {
+		return -ENOTTY;
+	}
+
+	const char *hostname = NULL;
+	const char *servname = NULL;
+	const struct addrinfo *hints = NULL;
+	struct addrinfo *res = NULL;
+	struct addrinfo *ai = NULL;
+
+	switch (in_arg->type) {
+	case GETADDRINFO:
+		hostname = in_arg->host_name;
+		servname = in_arg->serv_name;
+		hints = in_arg->ai_hint;
+		ret = lwip_getaddrinfo(hostname, servname, hints, &res);
+		if (ret != 0) {
+			printf("lwip_getaddrinfo() returned with the error code: %d\n", ret);
+			ret = ERROR;
```

Comment:
It is ioctl, you have to return one of value in errno.h , instead of ERROR....As I mentioned in previous comment.

Reply:
I see. I will choose one of values in errno.h.